rskj-core/src/main/java/co/rsk/mine/MinerServerImpl.java

```diff
@@ -604,7 +603,7 @@ public void buildBlockToMine(@Nonnull Block newBlockParent, boolean createCompet
             newBlockParent = blockchain.getBlockByHash(newBlockParent.getParentHash());
         }
 
-        logger.info("Starting block to mine from parent {} {}", newBlockParent.getNumber(), Hex.toHexString(newBlockParent.getHash()));
+        logger.info("Starting block to mine from parent {} {}", newBlockParent.getNumber() + " " + newBlockParent.getHash());
```

Comment:
Small bug on log here. Changed the two {} {} for a concatenation, but you left the others here. Just replace '+ " " +' with a comma.